tests/conftest.py

```diff
@@ -553,6 +553,31 @@ def get_markets():
             },
             'info': {},
         },
+        'NANO/USDT': {
```

Comment:
I think for the failing tests, the easiest is to move this pair to `shitcoinmarkets()`, which is an extension of this dictionary - but is only used for tests regarding pairlist. 

That way it won't break a multitude of other tests.

apart from that it's now flake8 that's breaking tests (run `flake8`) to see the bad lines without waiting for CI to kick in.